jdi-light-material-ui-tests/src/test/java/io/github/epam/material/tests/displaydata/ChipTests.java

```diff
@@ -0,0 +1,72 @@
+package io.github.epam.material.tests.displaydata;
+
+import io.github.epam.TestsInit;
+import org.testng.annotations.Test;
+
+import static io.github.com.StaticSite.*;
+import static org.hamcrest.Matchers.hasToString;
+
+public class ChipTests extends TestsInit {
+    @Test
+    public void defaultChipTest(){
+        displayDataDefaultChipPage.open();
+
+        basicCheck();
+        disabledCheck(); // first variant of class value
+        clickableCheck(3);
+    }
```

Comment:
need to add check for 'clickable link'
for this check we can assert the url which changes after click